src/ApiPlatform/Resources/ApiAccess.php

```diff
@@ -68,13 +68,10 @@
                 ],
             ],
             provider: QueryProvider::class,
-            extraProperties: [
-                'query' => GetApiAccessForEditing::class,
-                'CQRSQuery' => GetApiAccessForEditing::class,
-                'scopes' => ['api_access_read'],
-            ]
+            CQRSQuery: GetApiAccessForEditing::class,
```

Comment:
You can remove the `provider: QueryProvider::class,` when you use the `CQRSGet` it is already specified by default SAme suggestions for CQRS commands the processor is defined by default